wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -165,8 +165,8 @@ def populate_pod_spec(
             pod_spec["preemptionPolicy"] = resource_args.get("preemption_policy")
         if resource_args.get("node_name"):
             pod_spec["nodeName"] = resource_args.get("node_name")
-        if resource_args.get("node_selectors"):
-            pod_spec["nodeSelectors"] = resource_args.get("node_selectors")
+        if resource_args.get("node_selector"):
```

Comment:
Probably not worth doing, but a little utility method to copy over these values while doing the snake to camel change might improve readability.

Reply:
I've added a conversion from snake to camel utility for another use case that we can use here. But I think the kubernetes runner should be refactored such that we support every input argument for jobs, pods and containers without having them hard coded. We could optionally support these all as snake or camel case